src/DonationForms/DonationFormDataQuery.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Give\DonationForms;
+
+use Give\Donations\ValueObjects\DonationMetaKeys;
+use Give\Framework\QueryBuilder\JoinQueryBuilder;
+use Give\Framework\QueryBuilder\QueryBuilder;
+
+/**
+ * Class used for loading the number of donors, donations and revenue amounts for multiple forms
+ *
+ * @unreleased
+ */
+class DonationFormDataQuery extends QueryBuilder
```

Comment:
Let's add a Unit test for this, you can duplicate the one I started for Campaigns and replace with forms

https://github.com/impress-org/givewp/blob/ddf306c22d5def2f24dc11b7d2fd624cf7bd1498/tests/Unit/Campaigns/CampaignsDataQueryTest.php#L18

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7900/commits/a78d5c95d17e94202fb703d0f826eb03e9c8123d

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7900/commits/05058e84f07e38d9afd02929e8f42cdcc93d2faa